### PR TITLE
Add workbook isolation

### PR DESCRIPTION
**Pull request by imsitnikov**

### CODE REVIEW

File: src/components/error-response-presenter.ts [hunk 1]
```diff
@@ -643,6 +643,24 @@ export default (error: AppError | DBError) => {
                 },
             };
         }
+        case US_ERRORS.INCORRECT_WORKBOOK_ID_HEADER: {
+            return {
+                code: 400,
+                response: {
+                    code,
+                    message: 'Incorrect x-dl-workbookid header',
```
Comment (jhoncool) [inline src/components/error-response-presenter.ts:651]: Maybe `Workbook id header is incorrect`?

File: src/components/middlewares/resolve-workbook-id.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import {Request, Response, NextFunction} from '@gravity-ui/expresskit';
+import {AppError} from '@gravity-ui/nodekit';
+import {US_ERRORS, DL_WORKBOOK_ID_HEADER} from '../../const';
+import Utils from '../../utils';
+import {Feature, isEnabledFeature} from '../../components/features';
+
+export const resolveWorkbookId = async (req: Request, res: Response, next: NextFunction) => {
```
Comment (jhoncool) [inline src/components/middlewares/resolve-workbook-id.ts:7]: why async?

**Review by jhoncool — APPROVED** [review]

File: src/components/error-response-presenter.ts (hunk 1 above)
Comment (imsitnikov) [reply src/components/error-response-presenter.ts]: fixed

File: src/components/middlewares/resolve-workbook-id.ts (hunk 2 above)
Comment (imsitnikov) [reply src/components/middlewares/resolve-workbook-id.ts]: without any reason, a copypaste from other middlware.
fixed.

**Review by jhoncool — APPROVED** [review]

**Pull request merged**